### PR TITLE
Fix #500 Color is Null. Fix KeyWindow not correctly calculated

### LINKED ISSUE #500: [Bug] DrawingView crashes when Background property not set on iOS
### Description

When you try to export drawings from DrawingView with 
`DrawingViewControl.GetImageStream(DrawingViewControl.Width, DrawingViewControl.Width)`
Control crashes if "Background " property is not set

### Link to Reproduction Sample
[Sample.zip](https://github.com/CommunityToolkit/Maui/files/9142125/Sample.zip)
### Steps to Reproduce

1. Run the attached sample
2. Press "Show Popup" button
3. Draw something
4. Press "Done" button on Popup page

### Expected Behavior
App not crashing. Lines are exported 

### Actual Behavior
App crashes
### Basic Information

- Version with issue:

- Platform Target Frameworks: 
  - iOS: 

### Workaround
Set  "Background" property on DrawingView

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Description of Change ###

Fix NRE exceptions

 ### Linked Issues ###
 

 - Fixes #500 
 - Fixes NRE KeyWindow is null

 ### PR Checklist ###
 
 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs [hunk 1]
```diff
@@ -15,7 +15,6 @@ public class AlertView : UIView
 	/// Parent UIView
 	/// </summary>
 	public UIView ParentView => UIApplication.SharedApplication.ConnectedScenes.ToArray()
-		.Where(x => x.ActivationState == UISceneActivationState.ForegroundActive)
```
Comment (VladislavAntonyuk) [inline src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs:18]: this is redundant

File: src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs [hunk 2]
```diff
@@ -141,7 +141,7 @@ static void DrawBackground(CGContext context, Paint? brush, CGSize imageSize)
 		switch (brush)
 		{
 			case SolidPaint solidColorBrush:
-				context.SetFillColor(solidColorBrush.Color.ToCGColor());
+				context.SetFillColor(solidColorBrush.Color?.ToCGColor() ?? DrawingViewDefaults.BackgroundColor.AsCGColor());
```
Comment (VladislavAntonyuk) [inline src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs:144]: use DrawingView default background color if Background is not defined

**Review by brminnick — DISMISSED** [review]
Thanks Vlad!!!

File: src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs (hunk 2 above)
Comment (pictos) [reply src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs:144]: Is it ok on the other platofrms?

Comment (VladislavAntonyuk) [reply src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs:144]: that was the reason I added do-not-merge tag.
on android Extension method excepts nullable. so it works.
on Windows I added additional check

**Review by pictos — APPROVED** [review]

**Pull request merged**